packages/mui-material/src/Tabs/Tabs.js

```diff
@@ -634,7 +634,8 @@ const Tabs = React.forwardRef(function Tabs(inProps, ref) {
       scroll(nextScrollStart, { animation });
     } else if (tabMeta[end] > tabsMeta[end]) {
       // right side of button is out of view
-      const nextScrollStart = tabsMeta[scrollStart] + (tabMeta[end] - tabsMeta[end]);
+      const nextScrollStart =
+        tabsMeta[scrollStart] + (tabMeta[end] - tabsMeta[end]) + tabMeta[size];
```

Comment:
Should we consider the `size` for left side button as well, in the earlier above if condition?